README.md

```diff
@@ -1,6 +1,9 @@
-# Mesos executor for Titus
+# Virtual-Kubelet executor for Titus
 
-Currently runs titus containers using Docker.
+[![Build status](https://badge.buildkite.com/378856785cd7805f1efad9b69086205f82ec69ac8ba18e9479.svg)](https://buildkite.com/netflix/titus-executor)
```

Comment:
Rebase maybe?

Reply:
I merged master into this branch. I'm a little confused by this comment.

Reply:
Oh I see, sorry I didn't read and was looking at individual commits and not the overall diff